<launch>
  <node pkg="uav_control" type="takeoff_node.py" name="takeoff_node" output="screen">
    <param name="target_height" value="50"/>
    <param name="alt_tolerance" value="0.2"/>
  </node>
</launch>
